### PR TITLE
New citizen engagement page

### LINKED ISSUE #2180: Create a Citizen engagement page layout
### Overview
We need to create a template page for the program area Citizen engagement so that people can find the project related to this program area.

### Action Items
Development
- [x] Develop the Donate page based on the design in [Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=4947%3A26584) in the pink rectangle
- [x] Do not link the page to the website yet- this should remain an unlinked page for now, until the new home page is finished
- [x] Add header image to [Credits](https://www.hackforla.org/credits/) page

### Resources/Instructions
[How to Create a new HfLA website page](https://github.com/hackforla/website/wiki/How-to-create-a-page-on-the-HfLA-website)
[HfLA Site Architecture](https://github.com/hackforla/website/wiki/Hack-for-LA's-Site-Architecture)
[Credit.yml file template](https://github.com/hackforla/website/wiki/credit.yml-file-template)

Design issue for the Citizen Engagement page https://github.com/hackforla/website/issues/1879

### PR DESCRIPTION
**Pull request by R-Tomas-Gonzalez**
Fixes #2180 

### What changes did you make and why did you make them ?

This is a developed webpage of the figma design for Citizen Engagement. We originally discussed using liquid to pull data to fill each card, however, there is not enough data to implement that task — specifically for the "problem", "solution", and "impact" sections of each card. 

At the moment, all cards are static. I have left detailed comments in both the html and css files, but please reach out to me with any questions. 

<details>
<summary>Full Page Screen Shot</summary>
citizen engagement page screen shot
</details>

### CODE REVIEW

**Review by adam-abundis — CHANGES_REQUESTED** [review]
@R-Tomas-Gonzalez Thank you so much for taking on this large project and delivering a visual prototype. Can I have you make 2 changes:
1. Can you move up the bullet points in your pull request under the title `What changes did you make and why did you make them ?`. This is formatting for the review team and project managers. 
2. Can you also add an image to the PR. This can be done with the [GoFullPage Chrome extension](https://chrome.google.com/webstore/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl?hl=en). I would spin up the Docker build and take a full-page screenshot. You can drop the image into the textbox and it will create a markdown version of the image (hosted by GitHub).

**If you want to go above and beyond:** you can move the markdown image like:

```
![](https://via.placeholder.com/150)
```



to an HTML `` and wrap that in a `<details>/<summary>` tag like so:

```
<details>
   <summary>Click here to see my picture</summary>
   <img src="https://via.placeholder.com/150" />
</details>
```

<details>
   <summary>Click here to see my picture</summary>
   
</details>

**Note**: the `<details>/<summary>` tags are a built-in Accordion element. As you can see, `<summary>` is what will be placed next to the arrow and anything between `<details>` is shown. We do this so we keep the page clean and less distracting.

Thank you again for all your work on this. I am going to approve this PR and make some issues integrating liquid logic into the page after the PR is properly formatted. Great work! ⭐ ⭐ ⭐

**Comment by R-Tomas-Gonzalez** [comment]
> @R-Tomas-Gonzalez Thank you so much for taking on this large project and delivering a visual prototype. Can I have you make 2 changes:
> 
> 1. Can you move up the bullet points in your pull request under the title `What changes did you make and why did you make them ?`. This is formatting for the review team and project managers.
> 2. Can you also add an image to the PR. This can be done with the [GoFullPage Chrome extension](https://chrome.google.com/webstore/detail/gofullpage-full-page-scre/fdpohaocaechififmbbbbbknoalclacl?hl=en). I would spin up the Docker build and take a full-page screenshot. You can drop the image into the textbox and it will create a markdown version of the image (hosted by GitHub).
> 
> **If you want to go above and beyond:** you can move the markdown image like:
> 
> ```
> 
> ```
> 
> 
> 
> to an HTML `` and wrap that in a `<details>/<summary>` tag like so:
> 
> ```
> <details>
>    <summary>Click here to see my picture</summary>
>    
> </details>
> ```
> 
> Click here to see my picture
> **Note**: the `<details>/<summary>` tags are a built-in Accordion element. As you can see, `<summary>` is what will be placed next to the arrow and anything between `<details>` is shown. We do this so we keep the page clean and less distracting.
> 
> Thank you again for all your work on this. I am going to approve this PR and make some issues integrating liquid logic into the page after the PR is properly formatted. Great work! ⭐ ⭐ ⭐

Thanks so much! I made the changes to the pr description. I need to mention that the "LA’s Sustainable Development Goals." needs to be linked in the third paragraph of the header. I have the link given to me by the designer -> " https://sdg.lamayor.org/about/global-goals" which was given to me after I made this pull request. I can add this and make the change, or we can create a new issue for this. Please let me know and I will adjust!

Thanks again!

File: pages/citizen-engagement.html [hunk 1]
```diff
@@ -0,0 +1,308 @@
+---
+layout: default
+title: Citizen Engagement
+---
```
Comment (adam-abundis) [inline pages/citizen-engagement.html:5]: @R-Tomas-Gonzalez Can you add a new line under line 3 (title: Citizen Engagement) and add the permalink property
`permalink: /citizen-engagement`

This is needed to make the page available within Jekyll

**Review by adam-abundis — CHANGES_REQUESTED** [review]
@R-Tomas-Gonzalez Thank you for your patience. I have made one more request so it can be accessed within the site. Once that is complete, I will approve this PR.

File: pages/citizen-engagement.html (hunk 1 above)
Comment (R-Tomas-Gonzalez) [reply pages/citizen-engagement.html:5]: Hey @abuna1985, 

I went ahead and added the permalink front matter. I also went ahead and added alt text for all images in the file as well. Please let me know if anything else is needed on my end! Thanks!

**Review by adam-abundis — APPROVED** [review]
@R-Tomas-Gonzalez Thank you again for all your help on this. I was able to see the updated changes. I will create a separate issue to implement project data and liquid template to make the projects dynamic.

Keep up the good work! ⭐  ⭐  ⭐

**Review by macho-catt — APPROVED** [review]
This looks good. Good job @R-Tomas-Gonzalez!

**Comment by macho-catt** [comment]
@R-Tomas-Gonzalez huge oversight on my part, after the merge it changed the CSS on the main page, the projects pages, and other pages, so I had to revert your PR to fix it. Basically, the main page page cards changed and became larger when they were not supposed to:

<details>
<summary>Original</summary>

original

</details>

<details>
<summary>Main page after PR</summary>

cards-are-bigger

</details>

And the project pages changed as well:

<details>
<summary>Original</summary>

original-projects

</details>

<details>
<summary>Projects after PR</summary>

projects-messed

</details>

And I believe that's because you updated the default classes in your file changes. Instead, you should add in a new class for this page specifically without editing the default classes. For example, you edit the .page-card class in citizen_engagement.scss, which actually overrides the page-card classes in other pages as well. What you need is to create a new class and add that new class along with .page-card, and edit the new class instead so the page cards in other pages don't get affected. Same goes with the other classes in the projects pages.

Does that make sense? Apologies again for missing this. Please open a new PR to address these, and reach out to me if you have questions. Thanks.

**Pull request merged**